.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - deploy
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build app
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.5
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.5'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools
```

Comment:
Upgrading `setuptools` resolved the build error we've been seeing in Travis. Reference: https://github.com/django-haystack/django-haystack/issues/1571#issuecomment-706947188